corehq/ex-submodules/pillowtop/es_utils.py

```diff
@@ -71,7 +71,10 @@ def _get_analysis(*names):
 }
 
 # Allow removing settings from defaults by setting
-# the value to None in settings.ES_SETTINGS
+# the value to this constant in settings.ES_SETTINGS
+# TODO: change this constant to an object that isn't actually a meaningful
+# Elastic index settings value (`None` is how you revert a setting to its
```

Comment:
Thank you for the explanation. I understand the difference, but am unsure of the use case

>  The use case this could be important is if the commcare-cloud repo configures a cluster with specific "HQ default Elastic values", which may differ from the "Elastic default for that version". 

Settings values are set per environment in `commcare-cloud`, it would be an extremely confusing practice to explicitly set a value in `commcare-cloud` then unset in our hq settings (also configured in a different place in `commcare-cloud`), and I am not sure that is something we should be encouraging.

Reply:
Can you provide an example of the behavior you think is being encouraged? I'm not sure I follow.

In any case, there's no functional change here. I'll be sure to draw your attention to it if this changes in the future so you have an actual implementation to analyze.